theme/open/views/dataprivacy/privacy.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="section-imprint {{#unless loggedin}}loggedout{{/unless}}">
+	{{#unless loggedin}}
+		<h4>Datenschutzerklärung</h4>
+	{{/unless}}
+    <p>Hinweise zum Datenschutz entnehmen Sie bitte folgender Seite.</p>
+
+    {{#embed "dataprivacy/privacy_priv-und-daten"}}{{/embed}}
+
+    {{#embed "dataprivacy/privacy_verantw-stelle"}}{{/embed}}
+
+    {{#embed "dataprivacy/privacy_erh-persb-daten"}}{{/embed}}
+
+    {{#embed "dataprivacy/privacy_verarb-persb-daten"}}{{/embed}}
+
+	{{#embed "dataprivacy/privacy_newsletter"}}{{/embed}}
+
+    {{#embed "dataprivacy/privacy_cookies"}}{{/embed}}
```

Comment:
Einrückung Tab - Leerzeichen (einheitlich tab bitte)

Reply:
@adrianjost we have to lint the whole client project by the way...

Reply:
I know 🙄 But let's do it step by step and don't introduce new mixed indentation

Reply:
sure

Reply:
Vorerst korrigiere ich das händisch